docs/src/main/paradox/docs/fusion/architecture.md

```diff
@@ -2,30 +2,32 @@
 
-The build step produces a server artifact to run on a Node.js server, with client-side assets and Javascript.
+The build step produces a server artifact to run on a Node.js server, with client-side assets and JavaScript.
 
-We produce a docker image that is able to serve the compiled assets directly, which is available on
+We produce a Docker image that is able to serve the compiled assets directly, which is available on
 @link:[Dockerhub](https://hub.docker.com/repository/docker/bluebrain/nexus-web){ open=new }.
 
 Although the application is served by a Node.js server, the client communicates directly to @ref:[Nexus Delta](../delta/index.md) using @ref:[Nexus.js](../utilities/index.md#nexus-js).
 
 ## Pages
 
-Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing technologie and help seperating the application main features.
+Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing system and help separating the application main features.
 
-Pages serve a specific top level entity or functionality concerns of diverse users and activities, and provide varying access privileges to each. The main features that can be accessed from the home page are: @ref:[Organizations](../fusion/organizations.md), @ref:[Projects](../fusion/projects.md),  @ref:[Studios](../fusion/studios.md) and @ref:[My data](../fusion/my-data.md). 
+Pages serve a specific top level entity or functionality concerns of diverse users and activities, and provide varying access privileges to each. The main features that can be accessed from the home page are: @ref:[Organizations](../fusion/organizations.md), @ref:[Projects](../fusion/projects.md), @ref:[Studios](../fusion/studios.md) and @ref:[My data](../fusion/my-data.md).
 
 The page @ref:[Project](../fusion/project.md), is responsible for handling all aspects of managing a single project.
 
+@@@ note { .tip title="Change of search page" }
+
 The search page has been removed, but every element that was previously available on the search page now has a link on the home page. These links redirect users to the appropriate global search type page.
 
-The pages in Nexus Fusion are part of the source code and reside in the `src/pages` folder.
+@@@
 
-While all the previous feature is still available in the current version of Nexus Fusion, we expect its functionality to evolve and change in the upcoming release.
+The pages in Nexus Fusion are part of the source code and reside in the `src/pages` folder. While all the previous feature is still available in the current version of Nexus Fusion, we expect its functionality to evolve and change in the upcoming releases.
```

Comment:
"While all the previous **features are** still available..."

Reply:
"we expect **their** functionality to..."

Reply:
#4533